savant/deepstream/nvinfer/model.py

```diff
@@ -59,6 +59,55 @@ class NvInferModelType(Enum):
     """Custom model."""
 
 
+class NvInferScalingComputeHW(Enum):
+    """Enum for hardware to be used for scaling compute."""
+
+    AUTO = 0
+    """Auto select the hardware."""
+
+    GPU = 1
+    """Use GPU for scaling."""
+
+    VIC = 2
+    """Use VIC for scaling."""
+
+
+class NvInferScalingFilter(Enum):
+    """Enum for algorithm to be used for scaling."""
+
+    NEAREST = 0
+    """Nearest neighbor interpolation."""
+
+    BILINEAR = 1
+    """Bilinear interpolation."""
+
+    GPU_CUBIC = 2
+    """GPU cubic interpolation."""
+
+    VIC_5_TAP = 2
+    """VIC-based 5-tap interpolation."""
+
+    GPU_SUPER = 3
+    """GPU super resolution."""
+
+    VIC_10_TAP = 3
+    """VIC-based 10-tap interpolation."""
+
+    LANCOZ = 4
```

Comment:
should be [LANZOS](https://docs.nvidia.com/metropolis/deepstream/4.0/dev-guide/DeepStream_Development_Guide/baggage/group__ee__buf__surf__transform.html#gga29da322f0c4f4ed65d19cc70471f54c0af16ae125b342d0014f931bb3b9677fca)

Reply:
or probably GPU_LANZOS?